### DISCUSSION TITLE
Debugging C/C++ using code-server

### DISCUSSION THREAD

**Discussion by sirmbom** [coder/code-server#6597] [Q&A]
I need some help. I want to run and debug C/C++ projects using code-sever but it is proving really difficult. I have tried installing extensions to help with the debugging but regardless, I keep receiving this particular error: _Cannot start debugging because no launch configuration has been provided._
![Screenshot_20231224-023913](https://github.com/coder/code-server/assets/113727083/d0c4cd0e-079a-4563-9977-ea71d09f0e8a)

**Comment by code-asher** [comment]
I have not tried this myself but some others users seem to have gotten things working: https://github.com/coder/code-server/issues/5764#issuecomment-1324961626
